src/libs/OptionsListUtils.js

```diff
@@ -266,7 +266,9 @@ function getParticipantNames(personalDetailList) {
     // `_.contains(Array, value)` for an Array with n members.
     const participantNames = new Set();
     _.each(personalDetailList, (participant) => {
-        // TODO: maybe get participant name by accountID in personalDetails??
+        if (participant.accountID) {
+            participantNames.add(participant.accountID.toString());
+        }
```

Comment:
Sorry I wasn't actually 100% sure if this would be necessary, I didn't look deeply into where we're using these participant names - do you know if this is helpful? I put "maybe" because maybe some places we don't actually want accountIDs to show up in the place where we have participant names?

Reply:
Hum, I think it might make sense if we want to search for a user taking into consideration the URL of the profile (it shows the ID now instead of the email). Not really sure if it's useful though

Reply:
Ok that's fair! I see now that you're right, it's all about searching - so let's keep your change for now 👍